### PR TITLE
[Refactor] Moved sections and new pages under Production, Faq

### PR DESCRIPTION
**Pull request by valentimarco**
Should solve issue #153 

Add also some todos...

### CODE REVIEW

File: mkdocs/faq/basic_info.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+#### Can I talk to the Cat in a language different from English?
+
+Of course, you can: just change the prompts in the Plugin folder accordingly, and take care not to mix languages to get best results.
+
+#### How can I know where the Cat gets the answers? I'd like to know if it's using the files I uploaded or if it's querying the configured LLM.
+
+Just open the console in your browser to check the logs there. At some point soon, this information will end up in the user interface, but at the moment is behind the scenes.
+
+#### I sent to the Cat some text and documents I want to get rid of, How can I do?
```
Comment (kodaline) [inline mkdocs/faq/basic_info.md:21]:
```suggestion
#### I sent to the Cat some text and documents I want to get rid of, How can I achieve it?
```

File: mkdocs/faq/basic_info.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+You can delete the `long_term_memory` folder and restart the Cat!
```
Comment (kodaline) [inline mkdocs/faq/basic_info.md:23]:
```suggestion
You can delete the `long_term_memory` folder and restart the Cat! Remember that you will lost everything!
```

File: mkdocs/faq/basic_info.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+#### Are the configured LLM APIs used to "instruct" the Cat with the documents I'm going to upload?
+
+That's not exactly how it works: basically when you ask something to the Cat, we pass to the configured LLM a prompt with your actual question + data that can be useful to answer that question. Data can be parts of your documents or chat history.  
+Please check our documentation for more details about how the Cat works for you.
+
+#### Can I talk to the Cat in a language different from English?
+
+Of course, you can: just change the prompts in the Plugin folder accordingly, and take care not to mix languages to get best results.
+
+#### How can I know where the Cat gets the answers? I'd like to know if it's using the files I uploaded or if it's querying the configured LLM.
```
Comment (kodaline) [inline mkdocs/faq/basic_info.md:17]:
```suggestion
#### How can I know where the Cat gets the answers from? I'd like to know if it's using the files I uploaded or if it's querying the configured LLM.
```

File: mkdocs/faq/basic_info.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+#### Can I insert a long article into the chat?
+
+Please avoid pasting long articles into the chat.  
+Use Rabbit Hole to upload long texts instead: just click on the attachment icon in the chat input widget and upload your file.
+
+#### Are the configured LLM APIs used to "instruct" the Cat with the documents I'm going to upload?
+
+That's not exactly how it works: basically when you ask something to the Cat, we pass to the configured LLM a prompt with your actual question + data that can be useful to answer that question. Data can be parts of your documents or chat history.  
+Please check our documentation for more details about how the Cat works for you.
```
Comment (kodaline) [inline mkdocs/faq/basic_info.md:11]:
```suggestion
Please check our documentation for more details about how the Cat internally works.
```

File: mkdocs/faq/basic_info.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# Basic Info
+
+#### Can I insert a long article into the chat?
+
+Please avoid pasting long articles into the chat.  
+Use Rabbit Hole to upload long texts instead: just click on the attachment icon in the chat input widget and upload your file.
+
+#### Are the configured LLM APIs used to "instruct" the Cat with the documents I'm going to upload?
+
+That's not exactly how it works: basically when you ask something to the Cat, we pass to the configured LLM a prompt with your actual question + data that can be useful to answer that question. Data can be parts of your documents or chat history.
```
Comment (kodaline) [inline mkdocs/faq/basic_info.md:10]:
```suggestion
That's not exactly how it works: basically when you ask something to the Cat, we pass to the configured LLM a prompt with your actual question + data that can be useful to answer that question. Data can be part of your documents or chat history.  
```

File: mkdocs/faq/basic_info.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
+# Basic Info
+
+#### Can I insert a long article into the chat?
+
+Please avoid pasting long articles into the chat.
```
Comment (kodaline) [inline mkdocs/faq/basic_info.md:5]:
```suggestion
Please avoid copy-pasting long articles into the chat.  
```

File: mkdocs/faq/customization.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
+# Customization
+
+#### I want to build my own plugin for the Cat: what should I know about licensing?
+
+Plugins are any license you wish, you can also sell them.
```
Comment (kodaline) [inline mkdocs/faq/customization.md:5]:
```suggestion
You can set set any licence you want and you are not limited to selling your plugin.
```

File: mkdocs/faq/customization.md [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+The Cat core is GPL3, meaning you are free to fork and go on your own, but you are forced to open source changes to the core.
+
+#### Port 1865 is not allowed by my operating system and/or firewall
+
+Change the port as you wish in the `.env` file.
+
+```text
+# Decide host and port for your Cat. Default will be localhost:1865
```
Comment (kodaline) [inline mkdocs/faq/customization.md:13]:
```suggestion
# Set HOST and PORT for your Cat. Default will be localhost:1865
```

File: mkdocs/faq/customization.md [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+CORE_HOST=localhost
+CORE_PORT=9000
+```
+
+#### Can I use a different vector database than Qdrant?
+At the moment, we don't provide any way to switch the vector database 😿 but it is planned for the future.
```
Comment (kodaline) [inline mkdocs/faq/customization.md:19]:
```suggestion
At this moment we don't provide any way to switch the vector database 😿 but it is planned for the future.
```

File: mkdocs/faq/general.md [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+Please try to ask in the support channel in our discord server, and remember this is all volunteers effort: be kind! :)
+
+#### Why the Cat does not default to some open LLM instead of ChatGPT or GPT-3?
+
+Our intention is not to depend on any specific LLM: the Cat does not have a preference about which LLM to use. Nonetheless, at the moment, OpenAI tools still provide the best results for your bucks.
+Decision is up to you.
+
+#### Are text and documents sent to the Cat safe and not shared with anybody?
+
+Well, the local memory is safe and under your control, although embeddings and prompts are shared with your configured LLM, meaning you need to check how safe is the LLM.
```
Comment (kodaline) [inline mkdocs/faq/general.md:15]:
```suggestion
The local memory is safe and under your control, although embeddings and prompts are shared with your configured LLM, meaning you need to check how safe the LLM is.
```

File: mkdocs/framework/flows/rabbit-hole-ingestion.md [hunk 11]
```diff
@@ -0,0 +1,3 @@
+# Rabbit Hole Ingestion
+
+:construction_worker: Working in progress...
```
Comment (kodaline) [inline mkdocs/framework/flows/rabbit-hole-ingestion.md:3]:
```suggestion
:construction_worker: Work in progress...
```

File: mkdocs/plugins/examples.md [hunk 12]
```diff
@@ -0,0 +1,84 @@
+# Examples
+
+Here are listed a bunch of code-snippet
```
Comment (kodaline) [inline mkdocs/plugins/examples.md:3]:
```suggestion
Follows a bunch of code snippets
```

File: mkdocs/production/clientlib/community/clientlib-java.md [hunk 13]
```diff
@@ -0,0 +1,3 @@
+# &#128640; Client Library Java
+
+Working progress...
```
Comment (kodaline) [inline mkdocs/production/clientlib/community/clientlib-java.md:3]:
```suggestion
Work in progress...
```

File: mkdocs/production/use-cases/examples.md [hunk 14]
```diff
@@ -0,0 +1,3 @@
+# Examples
+
+#TODO: plx help. no idea
```
Comment (kodaline) [inline mkdocs/production/use-cases/examples.md:3]: LOL

**Review by kodaline — APPROVED** [review]
@valentimarco great work!
I just left some small suggestions

@pieroit if Marco wants to fix them ahah you can then merge it, for me it's fine

File: mkdocs/production/use-cases/examples.md (hunk 14 above)
Comment (pieroit) [reply mkdocs/production/use-cases/examples.md:3]: `plx` rulez

**Comment by valentimarco** [comment]
@pieroit @kodaline Ready to be merged!

**Comment by pieroit** [comment]
> @pieroit @kodaline Ready to be merged!

Awesome!

**Pull request merged**